workshops/models.py

```diff
@@ -97,11 +98,13 @@ class Person(models.Model):
     github     = models.CharField(max_length=STR_MED, unique=True, null=True)
     twitter    = models.CharField(max_length=STR_MED, unique=True, null=True)
     url        = models.CharField(max_length=STR_LONG, null=True)
-    slug       = models.CharField(max_length=STR_LONG, null=True)
+    username   = models.CharField(max_length=STR_MED, unique=True)
 
     # Set the custom manager
     objects = PersonManager()
 
+    USERNAME_FIELD = 'username'
```

Comment:
I got a bit further this morning, and spun off a blocking issue into
#160.  I'll continue once we've resolved that.